layer/CMakeLists.txt

```diff
@@ -46,22 +47,16 @@ endif()
 
 add_definitions(-DVK_ENABLE_BETA_EXTENSIONS)
 
-if (WIN32)
-    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_CRT_SECURE_NO_WARNINGS /bigobj")
```

Comment:
FWIW /bigobj looks like it was carried over from VVL. I'd actually like to fix it properly in VVL FWIW because when CPP files get too big it can cause problems/slowdowns in IDEs. Especially when running coding analysis tools.

Reply:
Ok I can remove it.

This said, it may be useful because we generated vulkan_profiles.hpp that is 2 MB but here the file remains quite small considering that it's generated using few Vulkan Profiles. But a user could include a lot more profiles so we may hit this case.